client/src/test/diagnostics.test.ts

```diff
@@ -53,6 +53,176 @@ suite('Should get diagnostics', () => {
 			{ message: 'no viable alternative at input \'2004-12-11T12\'', range: toRange(0, 11, 0, 13), severity: vscode.DiagnosticSeverity.Warning, source: 'ex' }
 		]);
 	});
+	test('blob check 1', async () => {
```

Comment:
Try to provide a description for the test that explains what is the use case. As it is not I have to go find the file `blob1.ion` and read its contents. 

Perhaps a better name would be `'blob invalid blob contents'` to indicate that the error is due to the blob having an incorrect value vs not closing curly braces correctly

Reply:
Sure, I will correct it later.